Tools/autotest/sim_vehicle.py

```diff
@@ -499,6 +499,8 @@ def run_in_terminal_window(autotest, name, cmd):
             progress("Cannot find %s process terminal" % name)
     else:
         p = subprocess.Popen(runme)
+        if under_macos():
```

Comment:
If DISPLAY isn't needed, can't the DISPLAY check above be removed? Seems like that was intended to run under macOS.